lib/hmmlearn/hmm.py

```diff
@@ -485,8 +489,9 @@ def _accumulate_sufficient_statistics(self, stats, X, lattice,
     def _do_mstep(self, stats):
         super()._do_mstep(stats)
         if 'e' in self.params:
-            self.emissionprob_ = (
-                stats['obs'] / stats['obs'].sum(axis=1, keepdims=True))
+            self.emissionprob_ = np.maximum(
+                self.emissionprob_prior - 1 + stats['obs'],
+                0) / stats['obs'].sum(axis=1, keepdims=True)
```

Comment:
Actually, doesn't the normalization need to be fixed here?

Reply:
Ah yeah, good catch